tests/e2e/utils/performance.ts

```diff
@@ -59,7 +65,7 @@ async function getLoadingDurations() {
  */
 export const logPerformanceResult = ( description, metrics ) => {
 	appendFileSync(
-		PERFORMANCE_REPORT_FILENAME,
+		performanceReportfilename,
 		JSON.stringify( { description, ...metrics } ) + '\n'
 	);
 };
```

Comment:
Could we type the function's parameters? Like this perhaps?

```suggestion
export const logPerformanceResult = (
	description: string,
	metrics: Record< string, unknown >
) => {
	appendFileSync(
		performanceReportfilename,
		JSON.stringify( { description, ...metrics } ) + '\n'
	);
};
```

Reply:
Definitely! I've updated the types on [58c4adf](https://github.com/Automattic/woocommerce-payments/pull/10352/commits/58c4adfad326d2e05d15d820d21d7aa9f5d06892).